CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for your interest in contributing to Grafonnet! We welcome all people
+who want to contribute in a healthy and constructive manner within our
+community. Grafonnet is developed within the Grafana community. Therefore we
+are following the same [Code of Conduct as
+Grafana](https://github.com/grafana/grafana/blob/master/CODE_OF_CONDUCT.md). To
+help us create a safe and positive community experience for all, we require all
+participants to adhere to it.
+
+## Plugin Scope
+
+**Grafonnet only aims to support official plugins.**
+
+We've drawn this line to ensure that the code base stays focused and manageable
+over time. We do, however, encourage development and use of community Grafonnet
+plugins. See the [Community
+Plugins](https://grafana.github.io/grafonnet-lib/community-plugins/) page for
+more info on this.
```

Comment:
```suggestion
While Grafonnet itself only supports core Grafana features, we strongly encourage development and use of community Grafonnet plugins. See the [Community Plugins](https://grafana.github.io/grafonnet-lib/community-plugins/) page for more info on this.
```

Reply:
Committed this change, but changed "Grafonnet plugins" to "Grafonnet extensions".